input/cql/HIVElements.cql

```diff
@@ -552,13 +552,10 @@ define "Facility-level testing Observation":
 @activity: HIV.B6 Capture or update client history
 @description: Client is currently pregnant
 */
-// TODO: Replace placeholder with relevant CQL logic
+
 define "Currently pregnant B.DE29":
-  exists "Currently pregnant B.DE29 Observation"
-define "Currently pregnant B.DE29 Observation":
-  [Observation: Concepts."Currently pregnant - HIV.B.DE29"] O
-    where O.status in { 'final', 'amended', 'corrected' }
-      and O.value is true
+  [Condition: Concepts."Currently pregnant - HIV.B.DE29"] C
+  sort by start of prevalenceInterval() 
```

Comment:
So, could we move the `sort` part here to the HIVIndicatorElements? I need something like this in the DT rules, but tied to Encounter.